webdriver/tests/bidi/input/perform_actions/invalid.py

```diff
@@ -432,7 +432,7 @@ async def test_params_pointer_action_move_coordinate_missing(
 
 
 @pytest.mark.parametrize("coordinate", ["x", "y"])
-@pytest.mark.parametrize("value", [None, "foo", True, 0.1, [], {}])
+@pytest.mark.parametrize("value", [None, "foo", True, [], {}])
```

Comment:
FYI as mentioned on the recently landed BiDi PR I'm updating the tests and that would have been a part as well.

Reply:
I did read too literary as `adding`, but also saw it was P3 on your end.